common/test/acceptance/tests/studio/test_studio_library.py

```diff
@@ -343,6 +343,65 @@ def test_delete_shifts_blocks(self):
         self.assertEqual(self.lib_page.xblocks[-1].name, '11')
         self.assertEqual(self.lib_page.get_page_number(), '1')
 
+    def test_previews(self):
+        """
+        Scenario: Ensure the user is able to hide previews of XBlocks.
+        Given that I have a library in Studio with 40 XBlocks
+        Then previews are visible
+        When I click the toggle previews button
+        The previews will not be visible
+        When I click the toggle previews button
+        The previews are visible
+        """
+        self.assertTrue(self.lib_page.are_previews_showing())
+        self.lib_page.toggle_previews()
+        self.assertFalse(self.lib_page.are_previews_showing())
+        self.lib_page.toggle_previews()
+        self.assertTrue(self.lib_page.are_previews_showing())
+
+    def test_previews_navigation(self):
+        """
+        Scenario: Ensure preview settings persist across navigation.
+        Given that I have a library in Studio with 40 XBlocks
+        Then previews are visible
+        When I click the toggle previews button
+        And click the next page button
+        The previews will not be visible
+        And the first XBlock will be the 11th one
+        And the last XBlock will be the 20st one
```

Comment:
Incredibly minor nit: If you're making edits anyways, this should be `20th` not `20st`